pom.xml

```diff
@@ -1467,11 +1466,15 @@
             <groupId>org.apache.avro</groupId>
             <artifactId>avro-mapred</artifactId>
           </exclusion>
-          <!--  this is needed and must be explicitly included later-->
+          <!--  Do not needed calcite if we disable hive.cbo.enable -->
```

Comment:
Maybe, the followings are more accurate with this PR?
- `Do not needed` -> `Do not need`
- `if we disable` -> `because we disabled`